### ISSUE TITLE
Inverse search in Zathura

### ISSUE THREAD

**Issue by merement** [SublimeText/LaTeXTools#762]
First and foremost, thank you, guys, for the great tool!

I'm not sure if this is an issue or an under-documented feature. For me (Linux Mint 18, Sublime 3, Zathura 0.3.5 all fresh installs, Sublime and Zathura are installed from the standard repository) inverse search didn't work until I've created *~/.config/zathura/zathurarc* (Zathura configuration file at the default place) with the line

    set synctex-editor-command 'subl %{input}:%{line}'

Forward search worked out of the box.

**Comment by ig0774** [comment]
Thanks for the bug report. It looks like I left out a bit of code in the forward sync part. I'll try to have a fix together soon.

**Comment by ig0774** [comment]
This is fixed in v3.8.4. Inverse search should now work out of the box.

**Comment by zxdawn** [comment]
I have the same problem with elementary OS 6 Odin, Sublime build 4126, Zathura 0.4.5, even if I edited the `~/.config/zathura/zathurarc`.

Here's the config file:
```
	"linux" : {
		"texpath" : "$PATH:/usr/texbin",
		"python": "/home/xin/miniconda3/bin/python",
		"sublime": "subl",
		"sync_wait": 1.5,
		"sublime_executable": "",
		"keep_focus_delay": 0.5
	},
```
